single-sentence-per-line-rule.js

```diff
@@ -0,0 +1,105 @@
+// @ts-check
+
+"use strict";
+
+// BUG: for some reason this rule does not detect sentences starting with *, _ or `
```

Comment:
I am not sure how to handle this. If anyone has any suggestions, feel free to contribute!